pkg/controller/controller.go

```diff
@@ -380,6 +380,12 @@ func (p *csiProvisioner) ProvisionExt(options controller.ProvisionOptions) (*v1.
 		}
 	}
 
+	if options.SelectedNode.Name != os.Getenv("NODE_NAME") {
```

Comment:
I would add a flag to this binary to control this behavior. This will break normal csi drivers where the provisioner can run on any node.

Reply:
I added a flag, but not sure about the name.